scripts/static-vdis

```diff
@@ -1,12 +1,12 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # Common functions for managing statically-attached (ie onboot, without xapi) VDIs
 
-from __future__ import print_function
-import sys, os, subprocess, json, urlparse, itertools
+
+import sys, os, subprocess, json, urllib.parse
```

Comment:
There are multiple import that do not work in python2, this was never properly tested

Reply:
If I understand correctly, after updating to python3, it can only run under Python 3.
I noticed the issue in [CA-388124](https://issues.citrite.net/browse/CA-388124), I'll sort it out ASAP.

Reply:
Sorry, my comment was misleading. the script is always run using python 3, as enforced by the shebang

Reply:
It's OK. If I were you, I would thought CA-388124 was caused by the python3 migration.